launchdarkly/README.md

```diff
@@ -1,45 +1,56 @@
 # LaunchDarkly Integration
 
-## Overview
+### Overview
```

Comment:
```suggestion
## Overview
```

Let's start with an H2 instead of an H3 to benefit from the automated ToC from Datadog Docs.